scripts/synthesis_cron.py

```
@@ -140,46 +507,33 @@ def run_synthesis(force: bool = False) -> int:
-        print(f"Synthesis complete for {date_label}")
-
-    if failed:
-        # Don't clear timestamp — some dates may have succeeded.
-        # Next run will re-extract only dates that still have pending sessions.
+    # Detect schema version and dispatch
+    conn = None
+    try:
+        from storage import get_db, close_db
+        conn = get_db()
+        version = _get_schema_version(conn)
+    except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)